### ISSUE TITLE
Shader failing on MacOSX Big Sur, IMAC20,2 , Radeon 5700XT 16Gb

### ISSUE THREAD

**Issue by pmacchiarella** [bnpr/Malt#57]
When I try to load the shader in the material tab I get this error:

bpy.ops.object.multires_subdivide(modifier="Multires")
bpy.ops.object.multires_subdivide(modifier="Multires")
bpy.ops.object.multires_subdivide(modifier="Multires")
bpy.ops.object.multires_subdivide(modifier="Multires")
bpy.ops.object.multires_subdivide(modifier="Multires")
bpy.ops.object.subdivision_set(level=5, relative=False)
bpy.context.space_data.context = 'TOOL'
bpy.context.space_data.context = 'RENDER'
bpy.context.scene.render.engine = 'MALT'
bpy.context.scene.view_settings.view_transform = 'Standard'
bpy.ops.outliner.item_activate(extend=False, deselect_all=True)
bpy.ops.sculpt.sculptmode_toggle()
bpy.ops.outliner.item_activate(extend=False, deselect_all=True)
bpy.context.space_data.context = 'DATA'
bpy.context.object.data.type = 'SUN'
bpy.context.space_data.context = 'MATERIAL'
bpy.context.space_data.system_bookmarks_active = 3
bpy.context.space_data.system_bookmarks_active = 7
NPR_Pipeline : PRE_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 19 samplers
NPR_Pipeline : MAIN_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 21 samplers
NPR_Pipeline : SHADOW_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 17 samplers

NPR_Pipeline : PRE_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 19 samplers
NPR_Pipeline : MAIN_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 21 samplers
NPR_Pipeline : SHADOW_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 17 samplers

NPR_Pipeline : PRE_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 19 samplers
NPR_Pipeline : MAIN_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 21 samplers
NPR_Pipeline : SHADOW_PASS : SHADER LINKER ERROR :
WARNING: Output of vertex shader 'MODEL' not read by fragment shader
ERROR: Implementation limit of 16 active fragment shader samplers (e.g., maximum number of supported image units) exceeded, fragment shader uses 17 samplers

I include my system info:
===========================================
= Blender 2.93.0 Alpha System Information =
===========================================

Blender:
===========================================

version: 2.93.0 Alpha, branch: master, commit date: 2021-02-03 23:01, hash: d85c5accecd5, type: Release
build date: 2021-02-04, 00:01:51
platform: Darwin
binary path: '/Applications/Blender.app/Contents/MacOS/Blender'
build cflags:  -Wall -Werror=implicit-function-declaration -Werror=return-type -Wno-tautological-compare -Wno-unknown-pragmas -Wno-char-subscripts -Wstrict-prototypes -Wmissing-prototypes -Wunused-parameter  -mmacosx-version-min=10.13 -Xclang -fopenmp -I'/Users/blender/blender-buildbot/macos_master/blender.git/../lib/darwin/openmp/include' -std=gnu11 -msse  -pipe -funsigned-char -fno-strict-aliasing -msse2  
build cxxflags:  -Wall -Wno-tautological-compare -Wno-unknown-pragmas -Wno-char-subscripts -Wno-overloaded-virtual -Wno-sign-compare -Wno-invalid-offsetof  -mmacosx-version-min=10.13 -ftemplate-depth=1024 -stdlib=libc++ -Xclang -fopenmp -I'/Users/blender/blender-buildbot/macos_master/blender.git/../lib/darwin/openmp/include' -std=c++17 -msse  -pipe -funsigned-char -fno-strict-aliasing -msse2  
build linkflags: -fexceptions -framework CoreServices -framework Foundation -framework IOKit -framework AppKit -framework Cocoa -framework Carbon -framework AudioUnit -framework AudioToolbox -framework CoreAudio -framework Metal -framework QuartzCore -framework ForceFeedback -liconv -Wl,-stack_size,0x100000 -Wl,-unexported_symbols_list,'/Users/blender/blender-buildbot/macos_master/blender.git/source/creator/osx_locals.map' -stdlib=libc++
build system: CMake

Python:
===========================================

version: 3.7.7 (default, Jul  8 2020, 14:54:17)  [Clang 11.0.0 (clang-1100.0.33.8)]
file system encoding: utf-8:surrogateescape
paths:
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/startup'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/modules'
	'/Applications/Blender.app/Contents/Resources/2.93/python/lib/python37.zip'
	'/Applications/Blender.app/Contents/Resources/2.93/python/lib/python3.7'
	'/Applications/Blender.app/Contents/Resources/2.93/python/lib/python3.7/lib-dynload'
	'/Applications/Blender.app/Contents/Resources/2.93/python/lib/python3.7/site-packages'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/freestyle/modules'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/addons/modules'
	'/Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/modules'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/addons'
	'/Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/addons_contrib'
	'/Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/BlenderMalt/MaltPath'
	'/Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/BlenderMalt/MaltDependencies'
	'/Users/pierluigi/Downloads/Shader-Examples'

Python (External Binary):
===========================================

binary path: '/Applications/Blender.app/Contents/Resources/2.93/python/bin/python3.7m'
version: Python 3.7.7

Directories:
===========================================

scripts:
	'/Applications/Blender.app/Contents/Resources/2.93/scripts/modules'
	'/Applications/Blender.app/Contents/Resources/2.93/scripts'
	'/Users/pierluigi/Library/Application Support/Blender/2.93/scripts'
user scripts: '/Users/pierluigi/Library/Application Support/Blender/2.93/scripts'
pref scripts: None
datafiles: '/Users/pierluigi/Library/Application Support/Blender/2.93/datafiles'
config: '/Users/pierluigi/Library/Application Support/Blender/2.93/config'
scripts : '/Users/pierluigi/Library/Application Support/Blender/2.93/scripts'
autosave: '/Users/pierluigi/Library/Application Support/Blender/2.93/autosave'
tempdir: '/var/folders/gx/kr7ngcz17qbf9q26df8m058h0000gn/T/blender_2i7yQN/'

FFmpeg:
===========================================

avcodec:   '58, 54, 100'
avdevice:  '58,  8, 100'
avformat:  '58, 29, 100'
avutil:    '56, 31, 100'
swscale:   ' 5,  5, 100'

SDL:
===========================================

Version: 2.0.12
Loading method: linked (WITH_SDL_DYNLOAD=OFF)

Other Libraries:
===========================================

OpenColorIO:  1,  1,  1
OpenImageIO:  2,  1, 15
OpenShadingLanguage:  1, 10, 10
OpenSubdiv:  0,  0,  0
OpenVDB:  7,  0,  0
Alembic:  1,  7, 12
USD:  0, 20,  5

OpenGL:
===========================================

renderer:	'AMD Radeon Pro 5700 XT OpenGL Engine'
vendor:		'ATI Technologies Inc.'
version:	'4.1 ATI-4.2.15'
extensions:
	GL_APPLE_client_storage
	GL_APPLE_container_object_shareable
	GL_APPLE_flush_render
	GL_APPLE_object_purgeable
	GL_APPLE_rgb_422
	GL_APPLE_row_bytes
	GL_APPLE_texture_range
	GL_ARB_ES2_compatibility
	GL_ARB_blend_func_extended
	GL_ARB_draw_buffers_blend
	GL_ARB_draw_indirect
	GL_ARB_explicit_attrib_location
	GL_ARB_gpu_shader5
	GL_ARB_gpu_shader_fp64
	GL_ARB_instanced_arrays
	GL_ARB_internalformat_query
	GL_ARB_occlusion_query2
	GL_ARB_sample_shading
	GL_ARB_sampler_objects
	GL_ARB_separate_shader_objects
	GL_ARB_shader_bit_encoding
	GL_ARB_shader_subroutine
	GL_ARB_shading_language_include
	GL_ARB_tessellation_shader
	GL_ARB_texture_buffer_object_rgb32
	GL_ARB_texture_cube_map_array
	GL_ARB_texture_gather
	GL_ARB_texture_query_lod
	GL_ARB_texture_rgb10_a2ui
	GL_ARB_texture_storage
	GL_ARB_texture_swizzle
	GL_ARB_timer_query
	GL_ARB_transform_feedback2
	GL_ARB_transform_feedback3
	GL_ARB_vertex_attrib_64bit
	GL_ARB_vertex_type_2_10_10_10_rev
	GL_ARB_viewport_array
	GL_ATI_texture_mirror_once
	GL_EXT_debug_label
	GL_EXT_debug_marker
	GL_EXT_depth_bounds_test
	GL_EXT_texture_compression_s3tc
	GL_EXT_texture_filter_anisotropic
	GL_EXT_texture_mirror_clamp
	GL_EXT_texture_sRGB_decode
	GL_NV_texture_barrier

Implementation Dependent OpenGL Limits:
===========================================

Maximum DrawElements Vertices:	1048575
Maximum DrawElements Indices:	150000

GLSL:
Maximum Varying Floats:	150000
Maximum Vertex Attributes:	16
Maximum Vertex Uniform Components:	4096
Maximum Fragment Uniform Components:	4096
Maximum Vertex Image Units:	16
Maximum Fragment Image Units:	16
Maximum Pipeline Image Units:	80

Cycles:
===========================================

CPU device capabilities: SSE2 SSE3 SSE41 AVX AVX2

Enabled add-ons:
===========================================

io_anim_bvh (version: (1, 0, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_anim_bvh/__init__.py)
io_curve_svg (version: UNKNOWN, path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_curve_svg/__init__.py)
io_mesh_ply (version: UNKNOWN, path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_mesh_ply/__init__.py)
io_mesh_stl (version: (1, 1, 3), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_mesh_stl/__init__.py)
io_mesh_uv_layout (version: (1, 1, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_mesh_uv_layout/__init__.py)
io_scene_fbx (version: (4, 22, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_scene_fbx/__init__.py)
io_scene_gltf2 (version: (1, 6, 3), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_scene_gltf2/__init__.py)
io_scene_obj (version: (3, 8, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_scene_obj/__init__.py)
io_scene_x3d (version: (2, 2, 5), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/io_scene_x3d/__init__.py)
cycles (version: UNKNOWN, path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/cycles/__init__.py)
blenderkit (version: (1, 0, 40), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/blenderkit/__init__.py)
measureit (version: (1, 8, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/measureit/__init__.py)
precision_drawing_tools (version: (1, 4, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/precision_drawing_tools/__init__.py)
btrace (version: (1, 2, 3), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/btrace/__init__.py)
curve_tools (version: (0, 4, 5), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/curve_tools/__init__.py)
add_curve_extra_objects (version: (0, 1, 4), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/add_curve_extra_objects/__init__.py)
archimesh (version: (1, 2, 2), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/archimesh/__init__.py)
archipack (version: (1, 2, 84), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/archipack/__init__.py)
add_mesh_BoltFactory (version: (0, 4, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/add_mesh_BoltFactory/__init__.py)
add_mesh_extra_objects (version: (0, 3, 7), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/add_mesh_extra_objects/__init__.py)
space_view3d_pie_menus (version: (1, 2, 9), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/space_view3d_pie_menus/__init__.py)
space_view3d_brush_menus (version: (1, 1, 8), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/space_view3d_brush_menus/__init__.py)
space_view3d_spacebar_menu (version: (1, 9, 4), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/space_view3d_spacebar_menu/__init__.py)
space_view3d_modifier_tools (version: (0, 2, 6), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/space_view3d_modifier_tools.py)
lighting_dynamic_sky (version: (1, 0, 6), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/lighting_dynamic_sky.py)
sun_position (version: (3, 1, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/sun_position/__init__.py)
lighting_tri_lights (version: (0, 1, 4), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/lighting_tri_lights.py)
materials_library_vx (version: (0, 6, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/materials_library_vx/__init__.py)
materials_utils (version: (2, 2, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/materials_utils/__init__.py)
object_print3d_utils (version: UNKNOWN, path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/object_print3d_utils/__init__.py)
mesh_auto_mirror (version: (2, 5, 3), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_auto_mirror.py)
mesh_bsurfaces (version: (1, 8, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_bsurfaces.py)
mesh_tools (version: (0, 3, 6), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_tools/__init__.py)
mesh_f2 (version: (1, 8, 4), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_f2.py)
mesh_looptools (version: (4, 7, 6), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_looptools.py)
mesh_snap_utilities_line (version: (5, 9, 19), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_snap_utilities_line/__init__.py)
mesh_tissue (version: (0, 3, 25), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_tissue/__init__.py)
mesh_tiny_cad (version: (1, 3, 2), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/mesh_tiny_cad/__init__.py)
node_wrangler (version: (3, 37), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/node_wrangler.py)
space_view3d_align_tools (version: (0, 3, 4), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/space_view3d_align_tools.py)
object_boolean_tools (version: (0, 4, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/object_boolean_tools.py)
object_carver (version: (1, 2, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/object_carver/__init__.py)
object_fracture_cell (version: (0, 2), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/object_fracture_cell/__init__.py)
greasepencil_tools (version: (1, 3, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/greasepencil_tools/__init__.py)
real_snow (version: (1, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/real_snow.py)
object_scatter (version: (0, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/object_scatter/__init__.py)
rigify (version: (0, 6, 2), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/rigify/__init__.py)
magic_uv (version: (6, 4, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/magic_uv/__init__.py)
mesh_easy_lattice (version: (1, 0, 1), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons_contrib/mesh_easy_lattice.py)
object_fracture_crack (version: (1, 0, 2), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons_contrib/object_fracture_crack/__init__.py)
automat (version: (0, 39), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons_contrib/automat/__init__.py)
render_to_print (version: (0, 3, 0), path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons_contrib/render_to_print.py)
B28_InstantMeshesRemesh (version: UNKNOWN, path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/B28_InstantMeshesRemesh.py)
Brush_Manager (version: (1, 2, 5), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/Brush_Manager/__init__.py)
EaZyMatZ (version: (1, 0, 0), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/EaZyMatZ/__init__.py)
jmesh-tools-master (version: (1, 4, 1, 0), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/jmesh-tools-master/__init__.py)
jsculpt-tools-master (version: (1, 0, 2, 1), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/jsculpt-tools-master/__init__.py)
OutlineHelper (version: (1, 1, 0), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/OutlineHelper/__init__.py)
speedretopo_2_9_v_0_0_8 (version: (0, 0, 8), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/speedretopo_2_9_v_0_0_8.py)
selector (version: (1, 0), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/selector.py)
SuperImageDenoiser_2_83 (version: (1, 4), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/SuperImageDenoiser_2_83.py)
auto_eye_v03 (version: (0, 3), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/auto_eye_v03/__init__.py)
BY-GEN (version: (0, 0, 6), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/BY-GEN/__init__.py)
logotracer-1-1 (version: ('1', '0'), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/logotracer-1-1/__init__.py)
BlenderMalt (version: UNKNOWN, path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/BlenderMalt/__init__.py)
Blender_light_manager (version: (1, 1, 3), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/Blender_light_manager.py)
animation_add_corrective_shape_key (version: UNKNOWN, path: /Applications/Blender.app/Contents/Resources/2.93/scripts/addons/animation_add_corrective_shape_key.py)
Kami_Manga_Hair_v1 (version: (1, 0), path: /Users/pierluigi/Library/Application Support/Blender/2.93/scripts/addons/Kami_Manga_Hair_v1.py)

**Comment by bjorn3** [comment]
Duplicate of https://github.com/bnpr/Malt/issues/28

tl;dr: Malt hits an implementation limit in the graphics driver.

**Comment by pmacchiarella** [comment]
Sorry, I didn’t see the report

**Comment by pragma37** [comment]
Deleting this, since we no longer provide Mac support.
